### PR TITLE
Hack in support for GitHub access tokens.

### PR DESCRIPTION
**Pull request by aerique**
It's not very pretty but I'm not going to spend more time on it today.

`git blame ./build-scripts/changelog-from-pr.py` > reviewers

Possible improvements:
- merge `if access_token …` occurences back into one assignment
- `var.get(attribute, None)` → `var.get(attribute)` ?

### CODE REVIEW

File: build-scripts/changelog-from-pr.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
                   help='Make one-lined changelog entries (for 4.0 and older)')
 argp.add_argument('--username',
                   help='Use the specified username for Basic Authentication to the GitHub API, allowing an higher rate limit')
```
Comment (Habbie) [inline build-scripts/changelog-from-pr.py]: s/an higher/a higher/ while you're here?

Comment (Habbie) [inline build-scripts/changelog-from-pr.py]:
```suggestion
                  help='Use the specified username for Basic Authentication to the GitHub API, allowing a higher rate limit')
```

**Comment by aerique** [comment]
> See comment posted earlier. Besides that, if this has been tested with and without a token, just merge it.

I don't think other credentials besides the token are possible anymore for GitHub.  I'd have to double-check that.

Without credentials still works but you'll run into rate limiting really quickly.

**Comment by Habbie** [comment]
> I don't think other credentials besides the token are possible anymore for GitHub. I'd have to double-check that.

"nothing" and "token" are the two ways I mean.


> Without credentials still works but you'll run into rate limiting really quickly.

Understood!

**Comment by aerique** [comment]
Ah right!  I'll give it a go later today since I need it for dstore and then I'll merge it.

**Pull request merged**